region/norfair/east/Purple Shaft.json

```diff
@@ -256,6 +257,36 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 2],
+      "name": "G-Mode",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        },
+        "comesThroughToilet": "any"
+      },
+      "requires": [
+        "h_HeatedGModeOpenDifferentDoor"
```

Comment:
These need to be off camera helpers because the camera doesn't follow when entering from the top.